datadog-siem-cloud/playbooks/playbooks.json

```diff
@@ -1326,15 +1326,15 @@
           "importedBy": [],
           "recordTags": [
             "Datadog",
-            "datadog-siem-cloud"
+            "datadog-cloud-siem"
           ]
         }
       ]
     }
   ],
   "exported_tags": [
     "Datadog",
-    "datadog-siem-cloud",
+    "datadog-cloud-siem",
```

Comment:
for data ingestion this tag will not work. This had to be same as connector name.